test/sharness/t0020-init.sh

```diff
@@ -19,7 +19,7 @@ test_expect_success "ipfs init fails" '
 	test_must_fail ipfs init 2> init_fail_out
 '
 
-test_expect_success "ipfs init output looks good" '
+test_expect_success POSIX "ipfs init output looks good" '
```

Comment:
ooof! we'll need an equivalent test for windows to ensure the lock is taken in a different way